### PR TITLE
Staging

### PR DESCRIPTION
**Pull request by joelwass**
org endpoints, enterprise github-auth, and github middlware for donations

### CODE REVIEW

File: api/organization/donation-delete.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-const { ORG_ROLES, MSGS: { INTERNAL_SERVER_ERROR, NO_DONATION, INSUFFICIENT_PERMISSIONS } } = require('../../helpers/constants')
+const { MSGS: { INTERNAL_SERVER_ERROR, NO_DONATION, INSUFFICIENT_PERMISSIONS } } = require('../../helpers/constants') // eslint-disable-line
```
Comment (stripedpajamas) [inline api/organization/donation-delete.js]: saddddddddddddd; nit: shouldn't have eslint disabled here. does not block merge

File: config/index.js [hunk 2]
```diff
@@ -118,6 +118,12 @@ class Config {
     return 500
   }
 
+  getGithubAppConfig () {
+    const id = this.env.github_app_id
+    const privateKey = Buffer.from(this.env.github_app_pem, 'base64').toString('utf8')
```
Comment (stripedpajamas) [inline config/index.js:123]: TODO: confirm these two env vars are in prod (can by copied from staging)

**Review by stripedpajamas — APPROVED** [review]

File: api/organization/donation-delete.js (hunk 1 above)
Comment (joelwass) [reply api/organization/donation-delete.js]: idk why eslint_disables are added. i don't do this manually, maybe when i run npx standard it's adding them?

File: config/index.js (hunk 2 above)
Comment (joelwass) [reply config/index.js:123]: rog

Comment (joelwass) [reply config/index.js:123]: added

**Review by stripedpajamas — APPROVED** [review]

**Pull request merged**